### PR TITLE
MoreRoomType

### PR DESCRIPTION
**Pull request by louis1706**

### CODE REVIEW

**Comment by VALERA771** [comment]
Can you write docs for new rooms?

**Comment by louis1706** [comment]
> Can you write docs for new rooms?

idk what to say for them

File: EXILED/Exiled.API/Features/Room.cs [hunk 1]
```diff
@@ -438,10 +438,16 @@ private static RoomType FindType(GameObject gameObject)
...
                 "HCZ_Servers" => RoomType.HczServers,
-                "HCZ_Room3" or "HCZ_Intersection_Junk" or "HCZ_Intersection" => RoomType.HczTCross,
+                "HCZ_Room3" => RoomType.HczTCross,
+                "HCZ_Intersection_Junk" => RoomType.HczIntersectionJunk,
+                "HCZ_Intersection" => RoomType.HczIntersectionJunk,
                 "HCZ_096" => RoomType.Hcz096,
-                "HCZ_Curve" or "HCZ_Corner_Deep" => RoomType.HczCurve,
-                "HCZ_Straight_C" or "HCZ_Straight" or "HCZ_Straight_PipeRoom" or "HCZ_Straight Variant" => RoomType.HczStraight,
+                "HCZ_Curve" => RoomType.HczCurve,
+                "HCZ_Corner_Deep" => RoomType.HczCurve,
```
Comment (Misfiy) [inline EXILED/Exiled.API/Features/Room.cs:446]: doesnt seem right to me...

Comment (louis1706) [reply EXILED/Exiled.API/Features/Room.cs:446]: > RoomType.HczCornerDeep ?

Oups yes

**Review by Thundermaker300 — APPROVED** [review]

**Pull request merged**